.github/workflows/build.yml

```diff
@@ -28,8 +28,11 @@ jobs:
       - name: Setup Gradle
         uses: gradle/gradle-build-action@v2
 
+      - name: Check
+        run: ./gradlew detekt
+
       - name: Build project
-        run: ./gradlew build -x detekt -x test
+        run: ./gradlew build
```

Comment:
this will run the `detekt` step two times, do we event need detekt on build instead of PR?

Reply:
I think its better to run detekt every time something changes in the code. We did the same in the json-logic-kmp project